package/kedro_viz/integrations/kedro/data_loader.py

```diff
@@ -114,3 +117,17 @@ def load_data(
 
     context = load_context(project_path=project_path, env=env)
     return context.catalog, context.pipelines
+
+def load_all_versions(filepath: str) -> Dict[str, any]:
```

Comment:
Bit nitpicky, but when reading the function name and looking at the return type, I wasn't sure what kind of data I would get back from the method. I'd personally call it `load_data_for_all_versions` to make it clear we actually load the data itself and not just the versions.